Buddies/Onboarding/SignUpInfoVC.swift

```diff
@@ -171,19 +171,28 @@ class SignUpInfoVC: LoginBase, UIImagePickerControllerDelegate, UINavigationCont
     
     @IBOutlet weak var bioText: UITextView!
     
+    func stringUntil(_ s: String?, _ c: Character) -> String? {
```

Comment:
could you add a comment explaining this function?

Reply:
I guess I get it - like "get the string up until the first time you see this character". Didn't see "get it" right away